cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/CruiseControlMetricsProcessor.java

```diff
@@ -535,9 +541,9 @@ private boolean enoughTopicPartitionMetrics(Cluster cluster, int brokerId) {
       boolean result = ((double) missingTopics.size() / topicsInBroker.size()) <= MAX_ALLOWED_MISSING_TOPIC_METRIC_PERCENT
           && ((double) missingPartitions.get() / cluster.partitionsForNode(brokerId).size() <= MAX_ALLOWED_MISSING_PARTITION_METRIC_PERCENT);
       if (!result) {
-        LOG.warn("Broker {} has {} missing topics metrics and {} missing partition metrics.",
-                 brokerId, missingTopics, missingPartitions);
-        LOG.trace("Missing topics: {}", missingTopics);
+        LOG.warn("Broker {} has {}/{} missing topics metrics and {}/{} missing partition metrics. Missing topics: {}. Missing partitions: {}.",
+                 brokerId, missingTopics.size(), topicsInBroker.size(), missingPartitions.get(), cluster.partitionsForNode(brokerId).size(),
+                 missingTopics, missingPartitions);
```

Comment:
Let's maybe move the missing partitions to a separate `trace` level log -- i.e. `LOG.trace("Missing partitions: {}", missingPartitions);`. The reason is that we have so many partitions that we do not want to keep in the `warn` log.